### DISCUSSION TITLE
Richesta di informazioni: inserimento dei moduli di evacuazione

### DISCUSSION THREAD

**Discussion by ghost** [iisgiua/giuaschool#614] [Richieste di aiuto] [answered]
Buon pomeriggio @trinko.
Volevo chiederle gentilmente se mi potesse illustrare come inserire i moduli di evacuazione, ed eventualmente, i moduli per i genitori e gli studenti.  Vorrei approfittarne per chiederle se sul vostro registro, fosse abilitata l'opzione che i docenti possano accedere ed utilizzare il moduli.
Grazie dell'attenzione, attendo un riscontro positivo.

**Comment by trinko** [comment, marked as answer]
Anzitutto, guarda la documentazione: [Uso dei moduli](https://iisgiua.github.io/giuaschool-docs/task-request.html)

**Reply by ghost** [reply]
Va bene, ora guardo!

**Reply by ghost** [reply]
Grazie, ora ho risolto tutto quanto! Non avevo notato questa pagina, grazie mille!
